src/Azure.Functions.Cli/Arm/ArmUriTemplates.cs

```diff
@@ -13,6 +13,7 @@ internal static class ArmUriTemplates
         public const string FunctionAppOnContainerAppsApiVersion = "2022-09-01";
         public const string ManagedEnvironmentApiVersion = "2022-10-01";
         public const string BasicAuthCheckApiVersion = "2022-03-01";
+        public const string FunctionsStacksApiVersion = "2022-03-01";
```

Comment:
To pull down the Stacks definitions, please use this API version: `2020-10-01`.

/cc @davidmrdavid @shimedh

Reply:
Changed the API and made the related changes because of that.